src/App.css

```diff
@@ -2,6 +2,13 @@
   position: relative;
   width: 1920px;
   height: 1080px;
-  background: linear-gradient(107.33deg, #0a0c38 0.21%, #117183 100.38%);
-  
+  background: linear-gradient(107.33deg, #0a0c38 0.21%, #117183 100.38%); 
 }
+.bulb{	
```

Comment:
Bulb is not part of App. Its part of a page so it should be present only within a page component css